proteus/cfemIntegrals.pyx

```diff
@@ -3995,7 +3997,9 @@ def updateGlobalJacobianFromElementJacobian_eb_CSR(np.ndarray elementNeighbors,
                                                    np.ndarray csrRowIndeces_ru,
                                                    np.ndarray csrColumnOffsets_eb_ru,
                                                    np.ndarray elementJacobian_eb,
-                                                   np.ndarray globalJacobian):
+                                                   globalJacobian):
```

Comment:
@manuel-quezada in addition to making sure the integer inputs to C are properly queried from the size of the input arrays, I also found some instances where the CSR Jacobian was declared as an numpy array. I think I caught most of these.